src/javascripts/cookies.js

```diff
@@ -0,0 +1,43 @@
+// Add cookie to session
+
+(function() {
+  var
+    cookie = false,
+    cookieContent = document.getElementById('cookie');
+
+  checkCookie();
+  if (cookie === true) {
+    cookieContent.style.display = 'none';
+  } else {
+    cookieContent.style.display = 'block';
+  }
+
+  function setCookie(cname, cvalue, exdays) {
+    var d = new Date();
+    d.setTime(d.getTime() + (exdays * 24 * 60 * 60 * 1000));
+    var expires = 'expires=' + d.toUTCString();
+    document.cookie = cname + '=' + cvalue + '; ' + expires;
+  }
+
+  function getCookie(cname) {
+    var name = cname + '=';
+    var ca = document.cookie.split(';');
+    for (var i = 0; i < ca.length; i++) {
+      var c = ca[i].trim();
+      if (c.indexOf(name) === 0) return c.substring(name.length, c.length);
+    }
+    return '';
+  }
+
+  function checkCookie() {
+    var cookieName = getCookie('UK_Parliament');
+    if (cookieName !== '') {
+      cookie = true;
+    } else {
+      if (cookieName !== '' && cookieName !== null) {
+        setCookie('UK_Parliament', 'disclaimerCookie', 14);
```

Comment:
UK_Parliament probably not the best name for the cookie - it's implied by path that it's set by UK Parliament. If someone else was to set the UK_Parliament cookie with a different value, it might end up breaking this...